mock-core-configs/mock-core-configs.spec

```diff
@@ -149,6 +149,9 @@ fi
 %ghost %config(noreplace,missingok) %{_sysconfdir}/mock/default.cfg
 
 %changelog
+* Mon Feb 17 2025 Adil Hussain <adil@linux.com> 42.1-1
+- configs: update distribution-gpg-keys v1.110
```

Comment:
Please do not add changelogs upstream; this is automatically added by Tito (generated from the git commit message) at the time of new release (v42.2).

Reply:
thanks